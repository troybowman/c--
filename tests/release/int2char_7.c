../real/int2char_7.c