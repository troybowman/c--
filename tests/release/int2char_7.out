x: @ 64
y: 123456 @
